src/v2/view-builder/views/IdentifierView.js

```diff
@@ -140,7 +153,19 @@ const Body = BaseForm.extend({
 
 export default BaseView.extend({
   Body,
-  Footer: IdentifierFooter,
+
+  initialize() {
+    // We override the default IdentifierFooter by passing in our custom shouldShowForgotPasswordLink() method.
+    this.Footer = IdentifierFooter.extend({
+      shouldShowForgotPasswordLink: () => {
+        // We don't add the forgot password link in the footer if SIW renders multi IDPs,
+        // instead in that case we add it before the IDP buttons in IdentifierView.
```

Comment:
....buttons in IdentifierView `as primary links`

Reply:
done